tests/datasets/test_schema.py

```diff
@@ -2,20 +2,22 @@
 
 
 def test_json_serialization():
-    s = Schema(
+    schema = Schema(
         feature_column_names=["feature_1"],
         embedding_feature_column_names={
             "embedding_feature": EmbeddingColumnNames(vector_column_name="embedding_vector")
         },
+        prompt_column_names=EmbeddingColumnNames(
+            vector_column_name="prompt_vector",
+            raw_data_column_name="prompt_text",
+        ),
+        response_column_names=EmbeddingColumnNames(
+            vector_column_name="response_vector",
+            raw_data_column_name="response_text",
+        ),
     )
 
     # serialize and deserialize.
-    p = s.to_json()
-    schema_from_json = Schema.from_json(p)
+    schema_from_json = Schema.from_json(schema.to_json())
 
-    assert schema_from_json.embedding_feature_column_names is not None
-    assert schema_from_json.embedding_feature_column_names["embedding_feature"] is not None
-    assert (
-        schema_from_json.embedding_feature_column_names["embedding_feature"].vector_column_name
-        == "embedding_vector"
-    )
```

Comment:
I would maintain some of the assertions. Just checking the json data is the same doesn't really check anything meaningful.

Reply:
nvm